text/0000-dual-light-shadow.md

```diff
@@ -129,6 +130,54 @@ If `"dual"` is used in this context, a compile-time error is thrown:
 </template>
 ```
 
+Only three values are accepted in this context: `"light"`, `"shadow"`, or `"inherit"`. Any other value will throw a compile-time error.
+
+### Inheritance
+
+`"inherit"` is a special value of `lwc:render-mode` that can only be used in the context of a child LWC component:
+
+```html
+<template lwc:render-mode="dual">
+    <x-component lwc:render-mode="inherit"></x-component>
+</template>
+```
+
+This instructs the `<x-component>` to render using whatever mode the parent component is using – either shadow or light. If `<x-component>` is not a dual-mode component, then a runtime error will be thrown.
+
+For the parent component, however, `"inherit"` can be used in non-dual-mode components as well as dual-mode components:
+
+```html
+<template lwc:render-mode="light">
+    <!-- x-component will render as light -->
+    <x-component lwc:render-mode="inherit"></x-component>

```

Comment:
two main comments:

1.
> `<x-component lwc:render-mode="inherit"></x-component>`
it could be `<x-component lwc:render-mode="dual"></x-component>` and it will work the same, it does the same as `inherit` but without introducing yet another token.

2.
it seems to me that this should be an error, if you're choosing the mode in the template, why using inherit down here? you should be specific, because you have chosen the context in which this mode will operate by doing it at the template level. you're not typing less, you're not gaining anything... it is now just more ambiguous for the developer to try to remember what mode the template is running (meaning scroll up).

Reply:
> it could be `<x-component lwc:render-mode="dual"></x-component>` and it will work the same, it does the same as inherit but without introducing yet another token.

We could do that, yeah. I don't have a strong opinion.

>  you're not typing less, you're not gaining anything

Are you saying that the "inherit" behavior should be implied by omitting the `lwc:render-mode` on the `<x-component>` altogether? This is fine, except that it introduces ambiguity on how a component behaves depending on the template it is rendered in. In a light/shadow template, it will be whatever default the component author (in this case the author of `<x-component>`) chose, whereas in a dual template, it will inherit from that template. To me, this is more confusing than making the omission of `lwc:render-mode` always mean "whatever default the component author chose."